### PR TITLE
[TO BE TESTED]Improve scheduled sc execution dest me test

### PR DESCRIPTION
**Pull request by iulianpascalau**

### CODE REVIEW

File: dataRetriever/requestHandlers/requestHandler.go [hunk 1]
```diff
@@ -196,7 +197,8 @@ func (rrh *resolverRequestHandler) RequestRewardTransactions(destShardID uint32,
 
 // RequestMiniBlock method asks for miniblock from the connected peers
 func (rrh *resolverRequestHandler) RequestMiniBlock(destShardID uint32, miniblockHash []byte) {
-	if !rrh.testIfRequestIsNeeded(miniblockHash) {
+	suffix := fmt.Sprintf("miniblocks_%d", destShardID)
```
Comment (sasurobert) [inline dataRetriever/requestHandlers/requestHandler.go:200]: I think smaller suffix would help.

Comment (sasurobert) [reply dataRetriever/requestHandlers/requestHandler.go:200]: Like 2 letters - and maybe define them as costs

Comment (iulianpascalau) [reply dataRetriever/requestHandlers/requestHandler.go:200]: on it

Comment (iulianpascalau) [reply dataRetriever/requestHandlers/requestHandler.go]: done

**Pull request merged**